python/lsst/ip/isr/isrStatistics.py

```diff
@@ -459,3 +470,34 @@ def measureProjectionStatistics(self, inputExp, overscans):
                 outputStats['AMP_VFFT_IMAG'][amp.getName()] = np.imag(verticalFFT).tolist()
 
         return outputStats
+
+    def copyCalibDistributionStatistics(self, inputExp, **kwargs):
+        """Copy calibration statistics for this exposure.
+
+        Parameters
+        ----------
+        inputExp : `lsst.afw.image.Exposure`
+            The exposure being processed.
+        **kwargs :
+            Keyword arguments with calibrations.
+
+        Returns
+        -------
+        outputStats : `dict` [`str`, [`dict` [`str`,`float]]
+            Dictionary of measurements, keyed by amplifier name and
+            statistics segment.
+        """
+        outputStats = {}
+
+        for amp in inputExp.getDetector():
+            ampStats = {}
+
+            for calibType in ('bias', 'dark', 'flat'):
+                if calibType in kwargs and kwargs[calibType] is not None:
```

Comment:
```suggestion
                if kwargs.get(calibType, None) is not None:
```

Reply:
Would either of these work if no keyword arguments are passed?

Reply:
Taking the more compact suggestion.  Even if nothing is passed, kwargs still is defined in this scope, so it should be fine.